### ISSUE TITLE
slash commands dont seem to be popping up...

### ISSUE THREAD

**Issue by HotPepa** [NullCode1337/NullRAT#31]
the slash commands for the bot dont seem to be popping up... any reasons why or what i did wrong?
![image](https://user-images.githubusercontent.com/81879455/163278955-160d1769-1c07-4a2b-98e9-3f4559c8a866.png)

**Comment by HotPepa** [comment]
![image](https://user-images.githubusercontent.com/81879455/163279782-edc914ef-cdbd-4f6d-a5e9-7710e36862cd.png)
might have found smthn. i have both installed but disnake and mss arent recognized

**Comment by usrDottik** [comment]
During the compilation time of NullRAT, did you set the Server ID correctly? 

The server ID is used to send the commands information to that 'guild' in Discord, which says the bot has those commands, check it again to see if it's wrong.

If you added the bot **BEFORE** executing it for at least the first time, don't expect **ANY** commands to show up, since they get updated and shown after the first start of NullRAT, in which the information on the slash commands is pushed onto that 'guild' when that happens.

TL;DR:

2 possible causes

1- You settled the incorrect server id in the compiler
2- You haven't run NullRAT for at least one time, so the command information hasn't been loaded into your Discord Server and no commands will display.

**Comment by NullCode1337** [comment]
> the slash commands for the bot dont seem to be popping up... any reasons why or what i did wrong? ![image](https://user-images.githubusercontent.com/81879455/163278955-160d1769-1c07-4a2b-98e9-3f4559c8a866.png)

reasons why it could happen
1) invalid bot invite link
2) server id is wrong

regardless, you can find proper steps for all of them in "Getting Variables.md"

**Comment by NullCode1337** [comment]
inactive.
reply if there are any more problems
